server/routes/students.js

```diff
@@ -99,7 +101,7 @@ router.post('/', async (req, res, next) => {
             student: student_instance
         });
     } catch (e) {
-        console.error("Error: students#one", e)
+        logger.error(`POST | action=student/ | email=${email} | error=${e}`)
```

Comment:
Looks to me like we might want a function to format the logs in this format you want? Maybe it could just take the request type/route from the request, the error, and optionally any additional metadata as an object?